fix typo (#922)
Consensys/doc.linea: docs/release-notes.mdx

  - `DataSubmittedV2` becomes `DataSubmittedV3`, `DataFinalized` becomes `DataFinalizedV3`
  (`DataFinalizedV2` was skipped to align version numbers).
  - The change also introduced gas optimizations for blob submission and finalization. 
- Create granular roles for contracts `LineaRollup`, `L2MessageService`, and `TokenBridge`:
  - `LineaRollup`: Introduce more granular pausing and an additional verifier unsetting role 
  - `L2MessageService`: Introduce more granular pausing
  - `TokenBridge`: Previously, all administration functions were managed by a single owner (multisig) 
  role. The update adds role-based access with more granular roles per function, particularly for 
  pausing.
- Remove `finalizeBlocksWithoutProof` function

>   - The function enabled finalization without proof. This allowed network activity to continue 

Reviewer: ```suggestion
  - The function enabled finalization without proofs. This allowed network activity to continue 
```

Adds support for Linea ENS domains, including a frontend app for users to register and manage their
domains. The Linea ENS system is comparable to ENS on Ethereum Mainnet, except that domains are 
limited to one per account, and registering requires completion of [Proof of Humanity](https://poh.linea.build/).

> As the system leverages CCIP Read, which enables L1 to seamlessly query the Linea ENS registry 

Reviewer: ```suggestion
As the system leverages CCIP Read, which enables L1 to trustlessly query the Linea ENS registry 
```

This would completely change the meaning -- `trustless` is also intended here

> through an off-chain gateway, Linea ENS domains resolve on L1. 

Reviewer: ```suggestion
through an offchain gateway, Linea ENS domains resolve on L1. 
```

    EIP-4844's arrival on Ethereum Mainnet enabled Ethereum L2s to start using a new transaction type 
    that transports L2 transaction and messaging data to L1 in "blobs". Blobs are a considerably 

>     more cost-efficient alternative to the existing method of posting L2 transactions and messaging data 

Reviewer: ```suggestion
    more cost-efficient alternative to the existing method of posting L2 transaction and messaging data 
```

`transaction data + messaging data`, rather than `transactions + messaging data`
